src/objectTranslations/Affiliation__c-fr.objectTranslation

```diff
@@ -1,96 +1,92 @@
-  </fields>
-  <fields>
-    <name>StartDate__c</name>
-    <label>Date de début</label>
-  </fields>
-  <fields>
-    <name>Status__c</name>
-    <label>Statut</label>
-    <picklistValues>
-      <masterLabel>Current</masterLabel>
-      <translation>Actuel</translation>
-    </picklistValues>
-    <picklistValues>
-      <masterLabel>Former</masterLabel>
-      <translation>Ancien</translation>
-    </picklistValues>
-  </fields>
-  <layouts>
-    <layout>HEDA Affiliation Layout</layout>
-    <sections>
-      <label>Informations sur l'affiliation</label>
-      <section>Affiliation Information</section>
-    </sections>
-    <sections>
-      <label>Liens personnalisés</label>
-      <section>Custom Links</section>
-    </sections>
-  </layouts>
-  <nameFieldLabel>Clé d'affiliation</nameFieldLabel>
-  <standardFields>
-    <name>Name</name>
-    <label>Clé d'affiliation</label>
-  </standardFields>
-  <gender>Feminine</gender>
-  <startsWith>Vowel</startsWith>
-  <caseValues>
-    <plural>false</plural>
-    <value>Affiliation</value>
-  </caseValues>
-  <caseValues>
-    <plural>true</plural>
-    <value>Affiliations</value>
-  </caseValues>
+    <caseValues>
+        <plural>false</plural>
+        <value>Affiliation</value>
+    </caseValues>
+    <caseValues>
+        <plural>true</plural>
+        <value>Affiliations</value>
+    </caseValues>
+    <fields>
+        <label>Organisation</label>
+        <name>Account__c</name>
+        <relationshipLabel>Contacts affiliés</relationshipLabel>
+    </fields>
+    <fields>
+        <label>Type d&apos;affiliation</label>
+        <name>Affiliation_Type__c</name>
+    </fields>
+    <fields>
+        <label>Contact</label>
+        <name>Contact__c</name>
+        <relationshipLabel>Comptes affiliés</relationshipLabel>
+    </fields>
+    <fields>
+        <label>Description</label>
+        <name>Description__c</name>
+    </fields>
+    <fields>
+        <label>Date de fin</label>
+        <name>EndDate__c</name>
+    </fields>
+    <fields>
+        <help>Si cette case est cochée, l’organisation de l’affiliation est stockée dans l’enregistrement du contact.</help>
+        <label>Principal</label>
+        <name>Primary__c</name>
+    </fields>
+    <fields>
+        <label>Rôle</label>
+        <name>Role__c</name>
+        <picklistValues>
+            <masterLabel>Applicant</masterLabel>
+            <translation>Candidat</translation>
+        </picklistValues>
+        <picklistValues>
+            <masterLabel>Employee</masterLabel>
+            <translation>Employé</translation>
+        </picklistValues>
+        <picklistValues>
+            <masterLabel>Faculty</masterLabel>
+            <translation>Professeur</translation>
+        </picklistValues>
+        <picklistValues>
+            <masterLabel>Prospect</masterLabel>
+            <translation>Prospect</translation>
+        </picklistValues>
+        <picklistValues>
+            <masterLabel>Student</masterLabel>
```

Comment:
@robertbwatson so I assume our org order by the masterLabel alphabetically then is "Student" is "Etudiant" in this file, but still appears after "Prospect"?

Reply:
I think that makes sense.